.circleci/config.yml

```diff
@@ -68,7 +68,7 @@ jobs:
             while true; do git reset --hard HEAD && git clean -fd && git lfs pull && break; done
       - save_cache:
           name: Store large files in cache
-          key: git-lfs-{{ .Branch }}
+          key: git-lfs-{{ .Branch }}-{{ epoch }}
```

Comment:
Should this `{{epoch}}` be merged to master? It would make the clone very heavy in size. lfs caché is content-addressed s there should not be any conflicts

Reply:
Same as below, but maybe we can pass of this epoch as the lfs overhead is quite light.